doc/nrf/gs_recommended_versions.rst

```diff
@@ -0,0 +1,187 @@
+           - Minimum version
+           - Toolchain manager version
+         * - CMake
+           - |cmake_min_ver|
+           - |cmake_recommended_ver_win10|
+         * - dtc
+           - |dtc_min_ver|
+           - |dtc_recommended_ver_win10|
+         * - git
+           -
+           - |git_recommended_ver_win10|
+         * - GNU Arm Embedded Toolchain
+           - |gnuarmemb_min_ver|
+           - |gnuarmemb_recommended_ver_win10|
+         * - gperf
+           - |gperf_min_ver|
+           - |gperf_recommended_ver_win10|
+         * - ninja
+           - |ninja_min_ver|
+           - |ninja_recommended_ver_win10|
+         * - Python
+           - |python_min_ver|
+           - |python_recommended_ver_win10|
+         * - SEGGER Embedded Studio (Nordic Edition)
+           - |ses_min_ver|
+           - |ses_recommended_ver_win10|
+
+   .. group-tab:: Linux
+
+      .. list-table::
+         :header-rows: 1
+
+         * - Tool
+           - Minimum version
+           - Tested version
+         * - ccache
+           -
+           - |ccache_recommended_ver_linux|
+         * - CMake
+           - |cmake_min_ver|
+           - |cmake_recommended_ver_linux|
+         * - dfu_util
+           -
+           - |dfu_util_recommended_ver_linux|
+         * - dtc
+           - |dtc_min_ver|
+           - |dtc_recommended_ver_linux|
+         * - git
+           -
+           - |git_recommended_ver_linux|
+         * - GNU Arm Embedded Toolchain
+           - |gnuarmemb_min_ver|
+           - |gnuarmemb_recommended_ver_linux|
+         * - gperf
+           - |gperf_min_ver|
+           - |gperf_recommended_ver_linux|
+         * - ninja
+           - |ninja_min_ver|
+           - |ninja_recommended_ver_linux|
+         * - Python
+           - |python_min_ver|
+           - |python_recommended_ver_linux|
+         * - SEGGER Embedded Studio (Nordic Edition)
+           - |ses_min_ver|
+           - |ses_recommended_ver_linux|
+
+   .. group-tab:: macOS
+
+      .. list-table::
+         :header-rows: 1
+
+         * - Tool
+           - Minimum version
+           - Toolchain manager version
+         * - CMake
+           - |cmake_min_ver|
+           - |cmake_recommended_ver_darwin|
+         * - dtc
+           - |dtc_min_ver|
+           - |dtc_recommended_ver_darwin|
+         * - git
+           -
+           - |git_recommended_ver_darwin|
+         * - GNU Arm Embedded Toolchain
+           - |gnuarmemb_min_ver|
+           - |gnuarmemb_recommended_ver_darwin|
+         * - gperf
+           - |gperf_min_ver|
+           - |gperf_recommended_ver_darwin|
+         * - ninja
+           - |ninja_min_ver|
+           - |ninja_recommended_ver_darwin|
+         * - Python
+           - |python_min_ver|
+           - |python_recommended_ver_darwin|
+         * - SEGGER Embedded Studio (Nordic Edition)
+           - |ses_min_ver|
+           - |ses_recommended_ver_darwin|
+
+
+Required Python dependencies
+############################
```

Comment:
```suggestion
***************************
```
This must be h2, not h1.
Note that the number of asterisks is probably wrong in the suggestion. ;)